.github/actions/cli-tests/action.yml

```diff
@@ -0,0 +1,72 @@
+name: 'ZKVyper CLI Tests'
+description: 'Executes zkvyper CLI tests'
+inputs:
+  use-prebuilt-zkvyper:
+    description: 'Whether to use prebuilt zkvyper executable (set to `true` if testing prebuilt binaries).'
+    required: false
+    default: 'false'
+  zkvyper-version:
+    description: 'Prebuilt version of zkvyper compiler to download and use in tests.'
+    required: true
+    default: '1.4.0'
+runs:
+  using: "composite"
+  steps:
+    - name: Download prebuilt zkvyper
+      shell: ${{ runner.os == 'Windows' && 'msys2 {0}' || 'bash' }}
+      if: inputs.use-prebuilt-zkvyper == 'true'
+      env:
+        ZKVYPER_DOWNLOAD_URL: "https://github.com/matter-labs/era-compiler-vyper/releases/download"
+        ZKVYPER_VERSION: ${{ inputs.zkvyper-version }}
+      run: |
+        OUTPUT=zkvyper
+        case "${RUNNER_OS}-${RUNNER_ARCH}" in
+          Linux-ARM64*)
+            ZKVYPER_DOWNLOAD_FILENAME="zkvyper-linux-arm64-v${ZKVYPER_VERSION}"
+            ;;
+          Linux-X64*)
+            ZKVYPER_DOWNLOAD_FILENAME="zkvyper-linux-amd64-v${ZKVYPER_VERSION}"
+            ;;
+          macOS-ARM64*)
+            ZKVYPER_DOWNLOAD_FILENAME="zkvyper-macosx-arm64-v${ZKVYPER_VERSION}"
+            ;;
+          macOS-X64*)
+            ZKVYPER_DOWNLOAD_FILENAME="zkvyper-macosx-amd64-v${ZKVYPER_VERSION}"
+            ;;
+          Windows*)
+            OUTPUT=zkvyper.exe
+            ZKVYPER_DOWNLOAD_FILENAME="zkvyper-windows-amd64-gnu-v${ZKVYPER_VERSION}.exe"
+            ;;
+        esac
+        curl --location -o $OUTPUT "${ZKVYPER_DOWNLOAD_URL}/${ZKVYPER_VERSION}/${ZKVYPER_DOWNLOAD_FILENAME}"
+        chmod a+x "${OUTPUT}"
+        echo "${PWD}" >> "${GITHUB_PATH}"
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
```

Comment:
Updated action to `v4` and `node-version` to 20.

Unfortunately, this will not fix annoying annotations about Node version, because lower node versions are in use in the actions. But I'll take care later when the underlying actions will release new versions.